pkg/apis/bundle/v1alpha1/object_template_types.go

```diff
@@ -23,26 +23,44 @@ import (
 type TemplateType string
 
 const (
-	// UndefinedTemplateType represents an undefined template type, and should
-	// default to GoTemplate.
+	// UndefinedTemplateType represents an undefined template type.
 	UndefinedTemplateType TemplateType = ""
 
 	// GoTemplate represents a go-template type.
-	GoTemplate TemplateType = "GoTemplate"
+	GoTemplate TemplateType = "go-template"
```

Comment:
To not pollute the package namespace, you might want to use a convention like:

```
TemplateTypeUndefined
TemplateTypeGo
TemplateTypeJsonnet
```

Reply:
Good idea. I'll come back and do this